### ISSUE TITLE
TypeError: port must be an integer

### ISSUE THREAD

**Issue by sydtrakked** [philosowaffle/plex-lifx-webhook#4]
I keep getting the following error:

![capture](https://user-images.githubusercontent.com/16075507/39284976-e27fb28e-48ca-11e8-9d54-578476354cc8.PNG)

**Comment by philosowaffle** [comment]
Please post a copy of your config.ini, need to verify that the `FlaskPort` setting is configured correctly.  Remember to remove sensitive information first such as API keys and player UUIDs.

However, I would recommend moving to the latest version of this script which is fully compatible with [Tautulli](https://github.com/Tautulli/Tautulli) and can be found [here](https://github.com/blacktwin/JBOPS/blob/master/fun/plex_lifx_color_theme.pyw).

**Comment by sydtrakked** [comment]
Below is the config. I changed the port because I believe I have another service using 5000. But I also changed the webhook port in my Plex server settings to the same. I have also left it at the default 5000 and I still get the same errors.

`[LOGGER]
LogFile: plex_lifx_webhook.log

[SERVER]
FlaskPort: 8282

[PLEX]
IgnorePlayerUUIDs:
LocalPlayersOnly: true

[LIFX]
APIKey: 
Brightness: .35
Duration: 3.0
NumColors: 5
ColorQuality: 10
DefaultPauseTheme: Sunset
DefaultPlayTheme: Deja Entendu
Lights: Ceiling, Lamp`
